### PR TITLE
feat(locale): added new method to set locale from JSON file

### PR DESCRIPTION
**Pull request by christyjacob4**

### CODE REVIEW

File: src/Locale/Locale.php [hunk 1]
```diff
@@ -36,6 +36,23 @@ static public function setLanguage(string $name, array $language): void //TODO a
         self::$language[$name] = $language;
     }
 
+    /**
+     * Set New Locale from JSON file
+     *
+     * @param string $name
+     * @param string $path
+     */
+    static public function setFromJSON(string $name, string $path): void
```
Comment (eldadfux) [inline src/Locale/Locale.php:45]: Let make this a bit more consistent with the previous method we had, maybe something like:
`setLanguageFromArray(string $code, array $translations)`
`setLanguageFromJSON(string $code, string $path)`

We should also add better documentation about the expected structure, and make sure the README covers all the relevant options.

File: .phpunit.result.cache [hunk 2]
```diff
@@ -0,0 +1 @@
+{"version":1,"defects":{"Utopia\\Tests\\LocaleTest::testTexts":4},"times":{"Utopia\\Tests\\LocaleTest::testTexts":0.002}}
```
Comment (eldadfux) [inline .phpunit.result.cache:1]: We should add this file to .gitignore

Comment (christyjacob4) [reply .phpunit.result.cache]: done :+1:

File: src/Locale/Locale.php (hunk 1 above)
Comment (christyjacob4) [reply src/Locale/Locale.php]: done :smile:

**Pull request merged**